server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -138,6 +138,7 @@ For more detail about how caches can be configured see <<tuningcache>>.
 ==== Communication details
 
 {project_name} uses multiple, separate clusters of Infinispan caches. Every {project_name} node is in the cluster with the other {project_name} nodes in same data center, but not with the {project_name} nodes in different data centers. A {project_name} node does not communicate directly with the {project_name} nodes from different data centers. {project_name} nodes use external {jdgserver_name} servers for communication across data centers. This is done using the link:https://infinispan.org/docs/11.0.x/titles/hotrod_java/hotrod_java.html[Infinispan Hot Rod protocol].
+>>>>>>> Stashed changes
```

Comment:
This shouldn't be here.

Reply:
@tkyjovsk Are you saying the entire preceding paragraph should be removed?